vendor/assets/javascripts/jquery.fullscreen.js

```diff
@@ -83,41 +73,16 @@
       return this;
     }
 
-    // You can potentially pas two arguments a color
-    // for the background and a callback function
-
-    var options = $.extend({
-        'background'      : '#111',
-        'callback'        : $.noop( ),
-        'fullscreenClass' : 'fullScreen'
-      }, props),
-
-      elem = this,
+    // You can pass a callback function as option
 
-    // This temporary div is the element that is
-    // actually going to be enlarged in full screen
-
-      fs = $('<div>', {
-        'css' : {
-          'overflow-y' : 'auto',
-          'background' : options.background,
-          'width'      : '100%',
-          'height'     : '100%'
-        }
-      })
-        .insertBefore(elem)
-        .append(elem);
-
-    // You can use the .fullScreen class to
-    // apply styling to your element
-    elem.addClass( options.fullscreenClass );
+    var elem = this;
 
     // Inserting our element in the temporary
     // div, after which we zoom it in fullscreen
 
-    requestFullScreen(fs.get(0));
+    requestFullScreen(elem.get(0));
 
-    fs.click(function(e){
+    elem.click(function(e){
```

Comment:
This was intended to close fullscreen, when the backdrop div is clicked. We no longer have a backdrop div. Please remove since is might lead to bugs.